### PR TITLE
Configurable import constant name

### LINKED ISSUE #18: Import constant name would be configurable
In my current application there is a namespace `Import` living `lib/import`. It has some historical roots and it's not easy to change it. This is unfortunately in conflict with `dry-rails` while booting and failed on `MyApp::Import` cannot be found.

## Examples

```
Dry::Rails.container do
  config.import_constant = 'MyImport'
end
```

## Resources

See discussion at https://twitter.com/deepj/status/1245084341920763909

### PR DESCRIPTION
**Pull request by diegotoral**
This is my attempt to close #18.

Please, let me know what you think about my changes and where it can be improved?

### CODE REVIEW

File: lib/dry/rails/container.rb [hunk 1]
```diff
@@ -41,6 +41,15 @@ class Container < System::Container
...
       setting :auto_register_paths, [].freeze, reader: true
 
+      # @overload config.import_constant=(import_constant)
+      #   Set a custom import constant name
+      #
+      #   @param import_constant [String]
+      #
+      #   @api public
+      # @!scope class
+      setting :import_constant, 'Import'.freeze, reader: true
```
Comment (solnic) [inline lib/dry/rails/container.rb:51]: @diegotoral let's call it `auto_inject_constant` and set its default value to `Deps` (that's how we called it in the upcoming Hanami 2.0)

**Comment by solnic** [comment]
@diegotoral thanks for renaming the setting, seems like now specs need to be updated to reflect that rename (doing `Import` => `Deps` in all spec files should be enough).

**Comment by diegotoral** [comment]
> @diegotoral thanks for renaming the setting, seems like now specs need to be updated to reflect that rename (doing `Import` => `Deps` in all spec files should be enough).

Oops! I think I forgot to commit the spec files. 🤔 I'll take a few minutes today to get it fixed.

**Comment by solnic** [comment]
@diegotoral notice that some specs started to fail in *master* so don't worry about them, I'll fix it.

**Comment by diegotoral** [comment]
@solnic Updated the branch to use the new `Deps` module. I also updated lines in docs that mentioned the old `Import` module.

**Comment by solnic** [comment]
@diegotoral thank you for this PR 😄 It's now merged and I'll release 0.2.0 as soon as I figure out why one spec is failing randomly now 🙄

**Comment by diegotoral** [comment]
It is a pleasure to help! 🖖

**Pull request merged**